.github/workflows/clean-ghcr.yaml

```
@@ -0,0 +1,32 @@
+---
+name: Delete Obsolete GHCR Images
+
+on:
+  workflow_call:
+    inputs:
+      image-names:
+        type: string
+        required: true
+        description: |
+          The names of the container images you want to delete old versions for.
```

A bit more to the point. (does it matter what I want? 😄)

```suggestion
          The names of the container images to delete old versions for.
```